.ci/magician/cmd/vcr_cassette_update.go

```diff
@@ -0,0 +1,210 @@
+package cmd
+
+import (
```

Comment:
@iyabchen qq: I assume this is the rewrite of https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/scripts/go-plus/vcr-cassette-update/vcr_cassette_update.sh. Where do we upload the cassettes to the GCS bucket after running in recording? Specifically [this step](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/scripts/go-plus/vcr-cassette-update/vcr_cassette_update.sh#L130) in the original script.

Reply:
Yes, this is for [vcr_cassette_update.sh](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/scripts/go-plus/vcr-cassette-update/vcr_cassette_update.sh).

Good catch, I completely miss that, just updated the code and added a test for recording.